src/pytest/test.py

```diff
@@ -96,113 +91,110 @@ def testAttributes(self):
         self.assertListEqual([0], res)
 
     def testUnion(self):
+        N = 100
+        r = self
+        self.assertOk(r.execute_command(
+            'ft.create', 'idx', 'schema', 'f', 'text'))
+        for i in range(N):
 
-        with self.redis() as r:
-            r.flushdb()
```

Comment:
I see you removed all flush before each test, this is not good in case we will like to run the tests on external database (like I do with enterprise)